src/core/network/client.ts

```diff
@@ -301,15 +296,11 @@ class ApiClient {
   callApi(method: string, data: { [k: string]: unknown } = {}): Promise<any> {
     const { token, options, response, responseEnd } = this
 
-    const payload = Object.keys(data)
-      .filter((key) => typeof data[key] !== 'undefined' && data[key] !== null)
-      .reduce(
-        (acc, key) => ({
-          ...acc,
-          [key]: data[key],
-        }),
-        {}
-      )
+    const payload = Object.fromEntries(
+      Object.keys(data)
+        .filter((key) => typeof data[key] !== 'undefined' && data[key] !== null)
+        .map((key) => [key, data[key]])
```

Comment:
Not bad, but can be improved:
```suggestion
      Object.entries(data)
        .filter(([key, value]) => value != null)
```

Reply:
Was wondering why you wanted `fromEntries` to appear there.